x/genutil/server/cli/gentx.go

```diff
@@ -44,7 +44,9 @@ var (
 
 // GenTxCmd builds the gaiad gentx command.
 // nolint: errcheck
-func GenTxCmd(ctx *server.Context, cdc *codec.Codec) *cobra.Command {
+func GenTxCmd(ctx *server.Context, cdc *codec.Codec, mbm sdk.ModuleBasicManager,
```

Comment:
```suggestion
func GenTxCmd(ctx *server.Context, cdc *codec.Codec,
```

Reply:
? the `mbm` is used though ? unresolve if I'm missing something